nncf/quantization/algo.py

```diff
@@ -359,14 +366,18 @@ def _quantize_activations(self, target_model: NNCFNetwork) -> List[InsertionComm
             original_nncf_graph = target_model.get_original_graph()
             for insertion_info, quantizer_config_list in insertion_data.items():
                 ia_op_exec_context = insertion_info.op_exec_context.input_agnostic
+                module_scope = str(ia_op_exec_context)
```

Comment:
This variable does not correspond to a scope of a module, or even to a string representation of a module scope.

Reply:
Done.